src/cca/islet_test/attest_seal_test.cc

```diff
@@ -0,0 +1,102 @@
+/*
+ *  Copyright (c) 2023 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the language governing permissions and
+ *  limitations under the License
+ */
+
+#include <islet.h>
+
+#include <iostream>
+#include <string>
+#include <cstring>
+
+#define BUFFER_SIZE 2048
+
+using byte = unsigned char;
+
+static const char CLAIM_TITLE_USER_DATA[] = "User data";
+static const char CLAIM_TITLE_PLATFORM_PROFILE[] = "Profile";
+
+bool attestation_test() {
+  byte report[BUFFER_SIZE];
+  byte claims[BUFFER_SIZE];
+  byte value[BUFFER_SIZE];
+  int report_len = 0;
+  int claims_len = 0;
+  int value_len = 0;
+
+  memset(report, 0, sizeof(report));
+  memset(claims, 0, sizeof(report));
```

Comment:
Although this will work, technically this should be : `sizeof(claims)`. Please fix.

Reply:
You're right. I fixed it.